packages/lib/src/Base.svelte

```diff
@@ -24,6 +25,7 @@
       -webkit-font-smoothing: antialiased;
       color: black;
       text-align: left;
+      display: contents;
```

Comment:
`display: contents` make sure that our `.base`-wrapper doesn't affect the dom (it makes it so that its own _space_ in the DOM is neglected, any children will still render as normal).

Reply:
Learn something new every day :)

Reply:
I also learnt something today! Or got reminded I should say.

`:host {}` is a much better way to deal with this. I removed the `<div class="base">` div and class in favour of this!